### PR TITLE
finished translating production_ready.md

### PR DESCRIPTION
**Pull request by chengat1314**

### CODE REVIEW

File: docs/ops/production_ready.md [hunk 1]
```diff
@@ -25,64 +25,53 @@ under the License.
...
+请注意，最大并行度必须满足以下条件： 
 
 `0 < parallelism  <= max parallelism <= 2^15`
 
-You can set the maximum parallelism by `setMaxParallelism(int maxparallelism)`. By default, Flink will choose the maximum
-parallelism as a function of the parallelism when the job is first started:
+可以通过 `setMaxParallelism（int maxparallelism)` 设置最大并行度。 
+默认情况下，Flink任务首次启动时会选择最大并行度作为并行性的函数: 
 
 - `128` : for all parallelism <= 128.
```
Comment (lviiii) [inline docs/ops/production_ready.md]: 以下原文也需要翻译喔：
- for all parallelism <= 128
- for all parallelism > 128

Comment (chengat1314) [inline docs/ops/production_ready.md]: ok, Sorry about that.

**Comment by chengat1314** [comment]
Please check again, thx!

**Comment by lviiii** [comment]
@chengat1314 thanks for your effort.
@tuhaihe please help to merge.

**Pull request merged**